File: Python/10.py
printf = print
printf("Esse programa determina o salario com base no reajuste e no FGTS!")

salario = float(input("\n\nSalario bruto: "))

if(salario<=2000):
    salario = (salario*15)/100
    printf(f"Seu Salario liquido é de R${salario}!")

elif(salario>2000 and salario<3000):
    salario = (salario*10)/100
    printf(f"Seu Salario liquido é de R${salario}!")

elif(salario>3000):
    salario=(salario*8)/100
    printf(f"Seu Salario liquido é de R${salario}!")

    printf("\n\nObrigado por usar nosso sistema!\n\n");

File: C/10.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <conio.h>
int main(void){
	setlocale(LC_ALL, "Portuguese");
	float salario;
	
	printf("Esse programa determina o salario com base no reajuste e no FGTS!");
	
	printf("\n\nSalario bruto: ");
	scanf("%f",&salario);
	
	if(salario<=2000){
		System.out.printf("Seu Salario liquido é de R$%.2f!",((salario*15)/100)+salario);	
	}
	if(salario>2000 && salario<3000){
		System.out.printf("Seu Salario liquido é de R$%.2f!",((salario*10)/100)+salario);	
	}
	if(salario>3000){
		System.out.printf("Seu Salario liquido é de R$%.2f!",((salario*8)/100)+salario);	
	}        

	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}

